MapboxNavigation/PollyVoiceController.swift

```diff
@@ -122,6 +122,13 @@ public class PollyVoiceController: RouteVoiceController {
     }
     
     func callSuperSpeak(_ text: String, error: String) {
+        guard let audioPlayer = audioPlayer else {
+            super.speak(fallbackText, error: error)
```

Comment:
https://github.com/mapbox/mapbox-navigation-ios/pull/624#discussion_r141154550 suggests canceling any existing Polly-related task before handing off to AVSpeechSynthesis, which sounds like a good idea.